analyzer/src/environment/variables.rs

```diff
@@ -77,10 +77,16 @@ impl Variables {
     /// This exposes their current state, which is only interesting for debugging.
     /// Use [`Variables::find_reachable`] to lookup any variable during the collection phase,
     /// or [`Variables::find_exported`] to lookup an exported variable after the collection phase.
+    #[cfg(debug_assertions)]
```

Comment:
```suggestion
```
This method is not for debug builds only, as it could also be used for visualization purposes.

Reply:
this is what documentation says

Reply:
The documentation states that is for debugging purposes, not for debug builds. Conditional compilation also is something to avoid, as integration tests need to treat the binaries as they are, and not recompile it.

Reply:
if it's only interesting for debugging then it's shall only be used when debugging. 
tests already compiles with debug_assertions set to true thus this not invalidates their build

Reply:
That's not the case. Execute `cargo test --release` to test with a release build, and the method will not be present, despite compiling the tests.